src/classes/OPP_OpportunityNaming_NEW_BATCH.cls

```diff
@@ -0,0 +1,100 @@
+/*
```

Comment:
Why the new class instead of just refactoring the existing class? I'm not sure we'd want to have both the old and new classes in the repo at the same time.

Reply:
I wasn't sure we were going to keep it, so I did it strictly as a proof.  I can replace the original with it, and eliminate the _NEW no problem!